.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4.1.1
-      
+      - uses: volta-cli/action@v4      
```

Comment:
One issue that we've been seeing sporadically with this approach, is that it'd sometimes fail saying that there was an exceeding number of requests coming from a Github worker IP address. I assume that this is somewhat luck based as the worker pool is shared with other users

Reply:
Yes, just noting that it's something that's been happening on practice. One potential improvement that we wanted to make is caching the volta-managed versions using the `cache` action, which would avoid having anything checked in as well